### PR TITLE
Addressing more of Gunter's comments

### PR DESCRIPTION
**Pull request by furry13**
Also adding 'ADVERTISE' message to the diagram

### CODE REVIEW

File: draft-ietf-dhc-addr-notification.md [hunk 1]
```diff
@@ -194,7 +194,8 @@ A server which is configured to support the address registration mechanism MUST
 
 ## DHCPv6 Address Registration Request Message
 
-The DHCPv6 client sends an ADDR-REG-INFORM message to inform that an IPv6 address is in use.  The format of the ADDR-REG-INFORM message is described as follows:
+The DHCPv6 client sends an ADDR-REG-INFORM message to inform that an IPv6 address is configured on the client's interface and can be used.
```
Comment (lcolitti) [inline draft-ietf-dhc-addr-notification.md:197]: Nit: maybe say it's "assigned to" the interface instead of "configured on"?

Comment (lcolitti) [reply draft-ietf-dhc-addr-notification.md:197]: ... and maybe remove "and can be used".

File: draft-ietf-dhc-addr-notification.md [hunk 2]
```diff
@@ -230,8 +231,8 @@ Unlike other types of messages, which are sent from the link-local address of th
 
 On clients with multiple interfaces, the client MUST only send the packet on the network interface that has the address being registered, even if it has multiple interfaces with different addresses. If the same address is configured on multiple interfaces, then the client MUST send ADDR-REG-INFORM each time the address is configured on an interface that did not previously have it, and refresh each registration independently from the others.
 
-The client MUST only send the ADDR-REG-INFORM message for valid ({{!RFC4862}}) addresses of global scope ({{!RFC4007}}). This includes ULA addresses, which are defined in {{!RFC4193}} to have global scope.
-The client MUST NOT send the  ADDR-REG-INFORM message for addresses configured by DHCPv6.
+The client MUST only send the ADDR-REG-INFORM message for valid ({{!RFC4862}}) addresses of global scope ({{!RFC4007}}). This includes ULA addresses, which are defined in {{!RFC4193}} to have global scope. This also includes statically assigned addresses of global scope (such addresses are considered to be valid indefinitely).
```
Comment (lcolitti) [inline draft-ietf-dhc-addr-notification.md:234]: I'm not sure it's a good idea to say addresses last forever....

**Review by lcolitti — APPROVED** [review]

File: draft-ietf-dhc-addr-notification.md (hunk 2 above)
Comment (furry13) [reply draft-ietf-dhc-addr-notification.md:234]: Let's sort it out tomorrow - I'm merging meantime.

**Pull request merged**